tensorboard/webapp/app_routing/route_config.ts

```diff
@@ -70,21 +77,68 @@ function getPathFragments(path: string): PathFragment[] {
   });
 }
 
+enum RouteType {
```

Comment:
Since this is not the 'type of a `Route`', in the sense of the `Route` type used throughout AppRouting, would it make sense to rename this to 'RouteDefType'?